docs/fides/docs/tutorial/google.md

```diff
@@ -87,7 +87,7 @@ The two declarations reflect the separate purposes for which data is collected a
 
 ## Check Your Progress
 
-After making the above changes, your app should resemble the state of the [`ethyca/fidesdemo` repository](https://github.com/ethyca/fidesdemo) at the [`fidesctl-add-google-analytics`](https://github.com/ethyca/fidesdemo/releases/tag/fidesctl-add-google-analytics) tag.
```

Comment:
@PSalant726 looking for your input here, i've updated the 2 places where there were broken links from the demo tutorial with this, i think linking to "releases" on fidesdemo is sufficient?

Reply:
Hm, yea it looks like the `add-google-analytics` tag was removed from the repo and broke this link. But, since we don't have any releases, and we're using tags to reference the progression through the tutorial, I think linking to the tags page (`https://github.com/ethyca/fidesdemo/tags`) is slightly better. We should change the copy as well so users aren't confused when they click through and the referenced tag isn't there. I'm going to pull in @NevilleS for his recommendation, since the GA tag was his idea and [I assume] he's the one who removed it.

Reply:
Oops- I think I just didn't finish updating the tags. I'll fix those all right now...

Reply:
Great, i'll change back, thanks Neville!